common/models/change.js

```diff
@@ -630,6 +637,74 @@ module.exports = function(Change) {
   };
 
   /**
+   * Resolve the conflict using the instance data in the source model.
+   *
+   * @callback {Function} callback
+   * @param {Error} err
+   */
+  Conflict.prototype.resolveUsingSource = function(cb) {
+    this.resolve(function(err) {
+      // don't forward any cb arguments from resolve()
+      cb(err);
+    });
+  };
+
+  /**
+   * Resolve the conflict using the instance data in the target model.
+   *
+   * @callback {Function} callback
+   * @param {Error} err
+   */
+  Conflict.prototype.resolveUsingTarget = function(cb) {
+    var conflict = this;
+
+    conflict.models(function(err, source, target) {
+      if (err) return done(err);
+      if (target === null) {
+        return conflict.SourceModel.deleteById(conflict.modelId, done);
+      }
+      var inst = new conflict.SourceModel(target);
```

Comment:
I just copied your code from the full-stack example, I am ok to use something else.

AFAICT, the difference between `save` and `updateAttributes` is in the way how undefined properties are handled. `updateAttributes` is ignoring undefined properties and preserving data in the database, while `save` will apply undefined properties in the database record too.